requirements-dev.txt

```diff
@@ -52,3 +52,4 @@ pydicom
 h5py
 nni
 optuna
+git+https://github.com/Project-MONAI/MetricsReloaded@monai-support#egg=MetricsReloaded
```

Comment:
@brudfors @wyli ,do we plan to release `MetricsReloaded` with versions? I think it's better to integrate with stable versions.

Thanks.

Reply:
I agree that pointing to a released version of MetricsReloaded would be prefered. I raised this with @csudre before the holiday break, maybe it is in the to-do list?